pydis_site/apps/api/viewsets/bot/reminder.py

```diff
@@ -126,3 +128,19 @@ class ReminderViewSet(
     queryset = Reminder.objects.prefetch_related('author')
     filter_backends = (DjangoFilterBackend, SearchFilter)
     filterset_fields = ('active', 'author__id')
+
+    def perform_destroy(self, instance: Reminder) -> None:
+        """Soft-delete reminders when DELETE is called."""
+        instance.active = False
+        instance.save()
+
+    def get_queryset(self) -> list[Reminder]:
+        """Filter out soft-deleted reminders by default."""
+        queryset = Reminder.objects.prefetch_related('author')
+        request: Request = self.request
+
+        include_active = request.query_params.get("include_active", "false")
+        include_active = include_active.lower() == "true"
+        if not include_active:
+            queryset = queryset.filter(active=True)
```

Comment:
shouldn't this be `include_inactive`?

Reply:
Yes, sorted in https://github.com/python-discord/site/compare/cedb0f1d724f4f2b9cb99d792331338695c3f7c3..52395f86fa3dd703372c878c739a225c366da98f